.github/workflows/ci.yml

```
@@ -66,14 +66,14 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-session-token: ${{ secrets.AWS_SESSION_TOKEN }}
-          role-to-assume: arn:aws:iam::037018655140:role/brightspace-ui-core.d2l.dev-githubactions-access
+          role-to-assume: arn:aws:iam::022062736489:role/r+BrightspaceUI+core+repo
           role-duration-seconds: 3600
           aws-region: ca-central-1
 
       - name: Publish
         uses: BrightspaceUI/actions/publish-to-s3@main
         with:
-          bucket-path: s3://brightspace-ui-core.d2l.dev/branches/pr-${{ github.event.number }}
+          bucket-path: s3://d2l.dev/live/prs/BrightspaceUI/core/pr-${{ github.event.number }}
```

Consider making this a little more copy-pastable

```suggestion
          bucket-path: s3://d2l.dev/live/prs/${{ github.repository }}/pr-${{ github.event.number }}
```

---

Good call Owen, I've updated [the docs](https://desire2learn.atlassian.net/wiki/spaces/DEVCENTRAL/pages/3493560351/How+To+Host+a+Static+Site+on+d2l.dev) to use `{{ github.repository }}` too!